exercises/practice/rotational-cipher/.meta/example.cpp

```diff
@@ -0,0 +1,22 @@
+#include "rotational_cipher.h"
+
+#include <sstream>
+
+namespace rotational_cipher {
+    std::string  rotate(const std::string& text, int shiftKey) {
```

Comment:
style nitpick: are the two spaces between return type and function name intentional? (in the .h file, too)